test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Random.Tests.ps1

```diff
@@ -156,6 +156,18 @@ Describe "Get-Random" -Tags "CI" {
         $randomNumber[6] | Should -BeNullOrEmpty
     }
 
+    It "Should return all the numbers for array of 1,2,3,5,8,13 in randomized order when the Shuffle switch is used" {
+        $randomNumber = Get-Random -InputObject 1, 2, 3, 5, 8, 13 -Shuffle
+        $randomNumber.Count | Should -Be 6
+        $randomNumber[0] | Should -BeIn 1, 2, 3, 5, 8, 13
+        $randomNumber[1] | Should -BeIn 1, 2, 3, 5, 8, 13
+        $randomNumber[2] | Should -BeIn 1, 2, 3, 5, 8, 13
+        $randomNumber[3] | Should -BeIn 1, 2, 3, 5, 8, 13
+        $randomNumber[4] | Should -BeIn 1, 2, 3, 5, 8, 13
+        $randomNumber[5] | Should -BeIn 1, 2, 3, 5, 8, 13
```

Comment:
From memory you can shorten these tests to just `$randomNumber | Should -BeIn 1, 2, 3, 5, 8, 13`